Main.java

```diff
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("hello");
+        int[] numbers = {1, 2, 3};
```

Comment:
The loop condition should be 'i < numbers.length' to avoid ArrayIndexOutOfBoundsException.